### ISSUE TITLE
Need a proper error message when `x-mgw-production-endpoints` is not defined

### ISSUE THREAD

**Issue by bhathiya** [wso2/product-microgateway#531]
**Description:**

This is what it logs when `x-mgw-basePath` is not defined.

```
micro-gw: 'x-mgw-basePath' property is not included in openAPI definition '/data/products/mg/test-project/api_definitions/petstore.yaml'.
``` 

But, when `x-mgw-production-endpoints` is not defined, this is the error logged.

```
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:108:34: undefined symbol 'get_e6e3e3e9_7d70_4d9d_88a7_3e1858443bd2_sand'
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:108:34: invalid remote function invocation, expected an client object
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:196:34: undefined symbol 'post_e4a00f91_66cf_48e7_90fb_ccc444512c7f_sand'
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:196:34: invalid remote function invocation, expected an client object
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:284:34: undefined symbol 'get_d9defaf1_9d8d_4757_9d24_12cd2847063d_sand'
error: src:0.0.0::Swagger_Petstore_1_0_0.bal:284:34: invalid remote function invocation, expected an client object
``` 

It's better to give a proper error message.

**Comment by praminda** [comment]
@VirajSalaka is working on a fix

**Comment by praminda** [comment]
Fixed by #534
